package.json

```diff
@@ -11,7 +11,7 @@
     ]
   },
   "scripts": {
-    "test": "set NODE_ENV=test && nyc mocha src/tests/*.js --require @babel/register",
+    "test": "npm run build && nyc mocha dist/tests/*.js",
```

Comment:
nice work but you have to set the environments to test so  add the testing environment to the script